react/src/components/footer/components/SprkFooterAwards/SprkFooterAwards.test.js

```diff
@@ -58,7 +58,8 @@ describe('SprkFooterAwards:', () => {
     };
 
     const wrapper = mount(<SprkFooterAwards awards={awards} />);
-    expect(wrapper.find('a').length).toBe(3);
+    expect(wrapper.find('a').length).toBe(2);
+    expect(wrapper.find('button').length).toBe(1);
```

Comment:
`SprkFooterAwards` uses `SprkToggle` and this test broke when updating.

Reply:
@guern1kn do we need an issue to update SprkFooterAwards component too or is it all set?

Reply:
`SprkFooterAwards` utilizes `SprkToggle` so it will be updated when we finish this task. No need for a separate issue.

Reply:
Thats good new!